src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countOfCalls = 0;
+
+  function getSum(firstOperand) {
+    return function(secondOperand) {
+      countOfCalls++;
+
+      if (countOfCalls >= 4 && countOfCalls % 2 === 0) {
```

Comment:
```suggestion
      if (countOfCalls >maxCalls && countOfCalls % 2 === 0) {
```
Better to create var for max calls and use it)